db/migrate/20230816140637_create_account_invitation.rb

```diff
@@ -0,0 +1,15 @@
+class CreateAccountInvitation < ActiveRecord::Migration[6.1]
+  def change
+    create_table :account_invitation do |t|
+      t.references :user, null: false, foreign_key: true
+      t.string :email
+      t.string :name
+      t.string :token
+      t.references :account, null: false, foreign_key: true
+      t.integer :status
+
+      t.timestamps
+    end
+    add_index :account_invitation, :token
```

Comment:
I wanted to say that we need more indexes. But let's better leave this for a separate task. We will fix the performance when the time comes. Also, it will be more clear which indexes are needed. For example, probably it would make sense to add a composed index for token + user_id but we don't know for sure yet. We will analyze the query when it comes to performance solutions.

Reply:
@VladislavSokov that means I suggest not to add indexes for now, let's drop it